docs/manual-test-plan.md

```
@@ -200,6 +233,21 @@ When a chart renders fully offline (no `lookup`), `talm apply -i` runs through t
+### D3. Per-node prefix on the maintenance-connection warning
+
+On a multi-node insecure apply where every node hits the `ok=false` (maintenance) path, each per-node emission of the warning must carry the node identifier prefix so the operator can correlate which line came from which node:
+
+```bash
+/tmp/talm-safety apply -i \
+  --nodes 192.0.2.10,192.0.2.11,192.0.2.12 \
+  --endpoints 192.0.2.10,192.0.2.11,192.0.2.12 \
+  -f nodes/node0.yaml
+```
+
+Expected (per node): `node 192.0.2.10: talm: drift verification unavailable on maintenance connection`. The single-node case (empty `nodeID`, the implicit path) MUST still emit the bare `talm: drift verification unavailable on maintenance connection` line — no `node : ` garbage prefix.
```

markdownlint-cli2 (0.22.1)

[warning] Spaces inside code span elements

(MD038, no-space-in-code)